### ISSUE TITLE
TypeError: Cannot read property 'ECHO' of undefined

### ISSUE THREAD

**Issue by JavaCS3** [jerch/node-termios#2]
Example not work

```
tty.c_lflag &= ~(sym.ECHO | sym.ECHONL);
                     ^

TypeError: Cannot read property 'ECHO' of undefined
```

**Comment by jerch** [comment]
Woops, this is a left over from the older API. The low level symbols are now under `.native`. Thx for spotting this. :+1:

**Comment by jerch** [comment]
Fixed in d21012fd1359bda360e74b781cc658b403018ec4.
